CONTRIBUTING.md

```diff
@@ -0,0 +1,298 @@
+
+You can also add any other necessary parameters.
+
+If you add a new demo, provide autotesting support as well:
+- add demo launch parameters in [demos/tests/cases.py](demos/tests/cases.py)
+- prepare list of input images in [demos/tests/image_sequences.py](demos/tests/image_sequences.py)
+
+Update [demos' README.md](demos/README.md) adding your demo to the list.
+
+## Accuracy Validation
+
+Accuracy validation can be performed by the [Accuracy Checker](./tools/accuracy_checker) tool. This tool can use either IE to run a converted model, or an original framework to run an original model. Accuracy Checker supports lots of datasets, metrics and preprocessing options, what simplifies validation if a task is supported by the tool. You only need to create a configuration file that contains necessary parameters for accuracy validation (specify a dataset and annotation, pre- and post-processing parameters, accuracy metrics to compute and so on). For details, refer to [Testing new models](./tools/accuracy_checker#testing-new-models).
+
+If a model uses a dataset which is not supported by the Accuracy Checker, you also must provide the license and the link to it and mention it in the PR description. 
+
+When the configuration file is ready, you must run the Accuracy Checker to obtain metric results. If they match your results, that means conversion was successful and the Accuracy Checker fully supports your model, metric and dataset. Otherwise, recheck the[conversion](#model-conversion) parameters or the validation configuration file.
+
+### Example
+
+This example uses one of the files from `tools/accuracy_checker/configs`  — validation configuration file for [AlexNet](tools/accuracy_checker/configs/alexnet.yml)\*:
+```
+models:
+  - name: alexnet-cf
+    launchers:
+      - framework: caffe
+        model:   public/alexnet/alexnet.prototxt
+        weights: public/alexnet/alexnet.caffemodel
+        adapter: classification
+    datasets:
+      - name: imagenet_1000_classes
+        preprocessing:
+          - type: resize
+            size: 256
+          - type: crop
+            size: 227
+          - type: normalization
+            mean: 104, 117, 123
+
+  - name: alexnet
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   public/alexnet/FP32/alexnet.xml
+        weights: public/alexnet/FP32/alexnet.bin
+        adapter: classification
+
+      - framework: dlsdk
+        tags:
+          - FP16
+        model:   public/alexnet/FP16/alexnet.xml
+        weights: public/alexnet/FP16/alexnet.bin
+        adapter: classification
+
+    datasets:
+      - name: imagenet_1000_classes
+        preprocessing:
+          - type: resize
+            size: 256
+          - type: crop
+            size: 227
+
+        metrics:
+          - name: accuracy@top1
+            type: accuracy
+            top_k: 1
+          - name: acciracy@top5
+            type: accuracy
+            top_k: 5
+```
```

Comment:
Again, no reason to copy the config here - just link to it.

Reply:
I would prefer to leave such examples in this document. They make it more understandable and and simply, because reduce "link clicks".

Reply:
For full understanding, I think it can be useful to use one model for all examples. It shows step by step what should be done. For me, if it will the same model which you use in model config demonstration is more suitable.

Reply:
Good point, thanks!